### PR TITLE
smp_utils: bump to version 0.99

### PR DESCRIPTION
**Pull request by grueni**

### CODE REVIEW

File: components/sysutils/smp_utils/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -52,37 +81,66 @@ file path=usr/bin/smp_zone_lock
...
-file path=usr/share/man/man1m/smp_zone_activate.1m
-file path=usr/share/man/man1m/smp_zone_lock.1m
-file path=usr/share/man/man1m/smp_zone_unlock.1m
-file path=usr/share/man/man1m/smp_zoned_broadcast.1m
+file path=usr/lib/$(MACH64)/libsmputils1.a
+link path=usr/lib/$(MACH64)/libsmputils1.so target=libsmputils1.so.1.0.0
+link path=usr/lib/$(MACH64)/libsmputils1.so.1 target=libsmputils1.so.1.0.0
+file path=usr/lib/$(MACH64)/libsmputils1.so.1.0.0
+file path=usr/sbin/$(MACH32)/smp_conf_general
+file path=usr/sbin/$(MACH32)/smp_conf_phy_event
```
Comment (AndWac) [inline components/sysutils/smp_utils/manifests/sample-manifest.p5m]: $(MACH32) doesn't look correct if we have BUILD_BITS= 64

Comment (grueni) [reply components/sysutils/smp_utils/manifests/sample-manifest.p5m]: Yes, I know.
I tried to use the options for configure but to no avail.
The installation procedure copies the same files to several directories.
The files in usr/bin are 64bit binaries and they are used in smp_utils.p5m.

**Comment by grueni** [comment]
Now it looks better.

**Comment by AndWac** [comment]
Please squash the commits.

**Pull request merged**